src/components/v2/VaultItemUi/styles.ts

```diff
@@ -0,0 +1,112 @@
+import { css } from '@emotion/react';
+import { useTheme } from '@mui/material';
+
+export const useStyles = () => {
+  const theme = useTheme();
+
+  return {
+    container: css`
+      width: 100%;
+    `,
+    header: css`
+      display: flex;
+      align-items: center;
+      justify-content: space-between;
+      margin-bottom: ${theme.spacing(6)};
+    `,
+    title: css`
+      display: flex;
+      align-items: center;
+    `,
+    tokenIcon: css`
+      width: 20px;
+      height: 20px;
+      margin-right: ${theme.spacing(1)};
+    `,
+    tokenIconLarge: css`
+      width: 32px;
+      height: 32px;
+    `,
+    tokenIconReward: css`
+      margin-left: ${theme.spacing(1)};
+    `,
+    text: css`
+      display: inline;
+    `,
+    textMobile14: css`
+      ${theme.breakpoints.down('sm')} {
+        font-size: 14px;
+      }
+    `,
+    textRewardValue: css`
+      font-weight: 600;
+    `,
+    textStakingValue: css`
+      display: inline-flex;
+      align-items: center;
+    `,
+    textAligned: css`
+      display: inline-flex;
+      align-items: center;
+    `,
+    rewardWrapper: css`
+      display: flex;
+      align-items: center;
+
+      ${theme.breakpoints.down('sm')} {
+        font-size: 14px;
```

Comment:
We can get this font size from the typography variants in the theme (`small1` or `small2` would work).

Reply:
this component has custom sizes that don't fit our typography styles.